packages/flutter_tools/lib/src/vscode/vscode.dart

```diff
@@ -35,7 +35,7 @@ class VsCode {
         .listSync()
         .where((FileSystemEntity d) => d is Directory)
         .where(
-            (FileSystemEntity d) => d.basename.startsWith(extensionIdentifier));
+            (FileSystemEntity d) => d.basename.toLowerCase().startsWith(extensionIdentifier.toLowerCase()));
```

Comment:
Perhaps make `extensionIdentifier` lowercase at it's definition?

```
static final String extensionIdentifier = 'Dart-Code.flutter'.toLowerCase();
```

Reply:
I originally did that, but then wondered if it'd be better to keep it close to where it's being checked so you didn't have to wonder "is this already lowercase?" when reading this code (and to protect you from someone copy/pasting over it in future - should it change again).

I don't have any strong preference for any, so let me know your preference and I'll tweak before merging.

Reply:
Ok, fixed this in a slightly different way... It occurred to me that we use the identifier elsewhere in the link - however, it turned out to be duplicated. So, I fixed that so we only have it in one place but that meant it didn't make sense to lowercase in the definition. So, now there's an additional variable - slightly weird but I think it solves a few issues (the `extensionIdentifier` isn't unexpected lower'd for other people, and the lowering is more obvious in the comparison code).